runelite-client/src/main/java/net/runelite/client/plugins/metronome/MetronomePluginConfiguration.java

```diff
@@ -36,21 +37,37 @@
 		keyName = "tickCount",
 		name = "Tick count",
 		description = "Configures the tick on which a sound will be played",
-		position = 2
+		position = 0
 	)
 	default int tickCount()
 	{
 		return 1;
 	}
 
+	@ConfigItem(
+		keyName = "tickVolume",
+		name = "Tick Volume",
+		description = "Configures the volume of the tick sound",
+		position = 1
+	)
+	default MetronomeVolume tickVolume() { return MetronomeVolume.MEDIUM_HIGH; }
```

Comment:
[The project's code conventions are to place curly braces on newlines.](https://github.com/runelite/runelite/wiki/Code-Conventions)
```suggestion
	default MetronomeVolume tickVolume()
	{
		return MetronomeVolume.MEDIUM_HIGH;
	}
```